core/src/main/java/org/elasticsearch/search/aggregations/metrics/avg/AvgAggregationBuilder.java

```diff
@@ -36,18 +37,19 @@
 import java.io.IOException;
 
 public class AvgAggregationBuilder extends ValuesSourceAggregationBuilder.LeafOnly<ValuesSource.Numeric, AvgAggregationBuilder> {
-    public static final String NAME = InternalAvg.TYPE.name();
+    public static final String NAME = "avg";
+    private final static Type TYPE = new Type("avg");
```

Comment:
Yeah, it should. I'll fix. I was checking to see if I could remove Type as part of this migration but I decided this was big enough as is.